examples/local/env.sh

```diff
@@ -56,12 +56,17 @@ if [ "${TOPO}" = "zk2" ]; then
     ZK_SERVER="localhost:21811,localhost:21812,localhost:21813"
     # shellcheck disable=SC2034
     TOPOLOGY_FLAGS="-topo_implementation zk2 -topo_global_server_address ${ZK_SERVER} -topo_global_root /vitess/global"
-
-    mkdir -p $VTDATAROOT/tmp
+elif [ "${TOPO}" = "k8s" ]; then
+    # Set topology environment parameters.
+    K8S_ADDR="localhost"
+    K8S_PORT="8443"
+    K8S_KUBECONFIG=$VTDATAROOT/tmp/k8s.kubeconfig
+    # shellcheck disable=SC2034
+    TOPOLOGY_FLAGS="-topo_implementation k8s -topo_k8s_kubeconfig ${K8S_KUBECONFIG} -topo_global_server_address ${K8S_ADDR}:${K8S_PORT} -topo_global_root /vitess/global"
 else
     ETCD_SERVER="localhost:2379"
     TOPOLOGY_FLAGS="-topo_implementation etcd2 -topo_global_server_address $ETCD_SERVER -topo_global_root /vitess/global"
-
-    mkdir -p "${VTDATAROOT}/tmp"
-    mkdir -p "${VTDATAROOT}/etcd"
```

Comment:
Should this have been deleted?

Reply:
Good catch. The etcd mkdir should still be in there. But the tmp dir creation is now done regardless of topology so it's not deleted, just moved.